src/plugins/kibana_react/public/toolbar_button/toolbar_button.scss

```diff
@@ -4,6 +4,10 @@
 
   // Some toolbar buttons are just icons, but EuiButton comes with margin and min-width that need to be removed
```

Comment:
Not critical, but a comment pointing to https://github.com/elastic/eui/issues/4730 would help us remember to replace/remove this code later.